corehq/apps/sms/models.py

```diff
@@ -1278,7 +1278,9 @@ def create_subevent_for_single_sms(self, recipient_doc_type=None,
             date=datetime.utcnow(),
             recipient_type=MessagingEvent.get_recipient_type_from_doc_type(recipient_doc_type),
             recipient_id=recipient_id,
-            content_type=MessagingEvent.CONTENT_SMS,
+            content_type=(MessagingEvent.CONTENT_CONNECT
```

Comment:
nit: it might be worth renaming this method since it is no longer an `sms` but generically a message.

`create_subevent_for_single_message` maybe?